# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qbankmanager/uic.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur qbankmanager-0.9.37-orig/src/kbanking/libs/dialogs/askaccount.ui qbankmanager-0.9.37/src/kbanking/libs/dialogs/askaccount.ui
--- qbankmanager-0.9.37-orig/src/kbanking/libs/dialogs/askaccount.ui	2006-03-08 14:08:16.000000000 +0000
+++ qbankmanager-0.9.37/src/kbanking/libs/dialogs/askaccount.ui	2006-03-22 08:50:00.350468750 +0000
@@ -320,6 +320,5 @@
 </tabstops>
 <layoutdefaults spacing="6" margin="11"/>
 <includehints>
-    <includehint>accountlistview.h</includehint>
 </includehints>
 </UI>
